src/app/test/dot-workflow-service.mock.ts

```diff
@@ -60,4 +60,8 @@ export class DotWorkflowServiceMock {
             }
         ]);
     }
+
+    fireWorkflowAction(): Observable<any[]> {
+        return Observable.of([]);
```

Comment:
we need return something here to test this case:

https://github.com/dotCMS/core-web/pull/582/files#diff-6a04a173805dcfecbbb2ca4256501928R34

Reply:
this method returns a contentlet, the test for the method itself of the dot-workflow-service.ts file is in dot-workflow.service.spec.ts